### ISSUE TITLE
temp and incoming folder permission

### ISSUE THREAD

**Issue by Bazzu85** [ngosang/docker-amule#23]
Hi,
running from Windows 11 on a Virtualbox Ubuntu WM.

I have the NTFS R: drive shared on the WM and the folder are writable.

here's my docker compose:
```
version: '3.3'
services:
    amule:
        image: ngosang/amule
        container_name: amule
        environment:
            - PUID=1000
            - PGID=1000
            - TZ=Europe/Rome
            - GUI_PWD=bazzucco04021985.cristian!
            - WEBUI_PWD=bazzucco04021985.cristian!
            - MOD_AUTO_RESTART_ENABLED=true
            - MOD_AUTO_RESTART_CRON=0 6 * * *
            - MOD_AUTO_SHARE_ENABLED=true
            - MOD_AUTO_SHARE_DIRECTORIES=/incoming;/my_movies
            - MOD_FIX_KAD_GRAPH_ENABLED=true
            - WEBUI_TEMPLATE=AmuleWebUI-Reloaded
        volumes:
            - ~/docker/amule/config:/home/amule/.aMule
            - /mnt/R/Downloads/aMule/incoming:/incoming
            - /mnt/R/Downloads/aMule/tmp:/temp
        network_mode: 'container:openvpn-client'
        restart: 'unless-stopped'
```
When starting amule I get this error for temp folder (same for incoming after)
```
2022-10-06 23:49:16: ERROR: Error accessing directory. Permissions on the temp directory too strict!
!2022-10-06 23:49:16: aMule cannot proceed. To fix this, you must set read/write/exec
!2022-10-06 23:49:16: permissions for the folder '/temp'
!2022-10-06 23:49:16: Attempting to use default directory at location 
!2022-10-06 23:49:16: '/home/amule/.aMule/Temp'.
```
trying to enter the container from terminal and making a dir on both temp and incoming folder works..

**Comment by ngosang** [comment]
I'm pretty sure this is not a bug in this container.
Said that, I don't have experience with Docker in Windows. If the folder `/mnt/R/Downloads/aMule/tmp` has enough privileges in the Windows host it should work.

**Comment by Bazzu85** [comment]
Hi, thx for the reply. A month is passed from the problem and in the meanwhile I abandoned the docker solution to a Hyper-V windows machine.
Using directly docker in windows I had many problems downloading files compairing to a normal emule installation, the same was on the ubuntu vm fixing the permission problems.

BTW From what I remember the problem was not from windows but from the user specified in the docker container. specifying a user in the virtualbox user group (found some guides on web) made it works but the files started downloading and after a minutes stops..
So I abandoned the whole docker environment
